request.go

```diff
@@ -15,16 +16,43 @@ import (
 // MaxFilelist is the max number of files to return in a readdir batch.
 var MaxFilelist int64 = 100
 
+// CheckFileRequest contains the data for a file hashing request
+type CheckFileRequest struct {
+	HashAlgoList string
+	StartOffset  uint64
+	Length       uint64
+	BlockSize    uint32
+}
+
+// GetHashAlgoAsList returns the comma separataed HashAlgoList string as slice
+func (r CheckFileRequest) GetHashAlgoAsList() []string {
+	result := []string{}
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Use `var result []string` to get an empty list.